README.md

```diff
@@ -67,6 +67,42 @@ Smoke tests expect an address of deployed application to be passed in `TEST_URL`
 ```
 
 By default, it will use `http://localhost:8581` which is defined in [src/smokeTest/resources/application.yaml](/src/smokeTest/resources/application.yaml).
+
+### Setting up API (gateway) tests
+
+Bulk Scan Processor uses an (Azure API Management) API to protect its SAS token dispensing endpoint.
+The API allows only HTTPS requests with a valid subscription key.
+
+Jenkins (pipeline) runs the API gateway tests by executing `apiGateway` gradle task. This only happens if
+there's a call to `enableApiGatewayTest()` in your Jenkinsfile_CNP/Jenkinsfile_parameterized.
+
+Your API tests rely on the following environment variables:
+- `TEST_CLIENT_SUBSCRIPTION_KEY` - subscription key that allows access to the API. Must be set manually in Azure Key Vault.
+- `API_GATEWAY_URL` - The URL of the API (gateway) that is the target of tests.
+This is provided by Jenkins based on the `api_gateway_url` output variable, defined in Terraform code.
```

Comment:
this is all done already, to we need to put it in readme?

Reply:
Not sure. When the subscription key expires or the service is recreated, the subscription key will have to be stored again. What do you think?